memcachepp/memcache/parser.hpp

```diff
@@ -1,6 +1,6 @@
 
 // Copyright 2007, 2008 (c) Friendster, Inc.
-// Copyright 2007-2010 (c) Dean Michael Berris <mikhailberis@gmail.com>
+// Copyright 2007, 2008 (c) Dean Michael Berris <dmberris@friendster.com>
```

Comment:
This change because i found the repository from svn://svn.code.sf.net/p/memcachepp/svn/trunk initially.